tests/integration/query/json/with_none_test.go

```diff
@@ -107,9 +107,6 @@ func TestQueryJSON_WithNoneFilterAndNestedArray_ShouldFilter(t *testing.T) {
 				Results: map[string]any{
 					"Users": []map[string]any{
 						{"name": "Shahzad"},
-						{"name": "John"},
-						{"name": "Islam"},
-						{"name": "Bruno"},
```

Comment:
thought:  This looks rather odd to me, although I was on the minority side in the standup discussion around this and got a little lost along the way.

@sourcenetwork/database-team Is this the behaviour we want?

Reply:
Discussed in standup, this is what the team wants :)